bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -127,6 +127,17 @@ private MergeSource(String repositoryName, String branchName) {
             if (!branchMatcher.matches()) {
                 return Optional.empty();
             }
+
+            // Verify that the branch exists
```

Comment:
```suggestion
            // Check if name refers to branch or repository
```

Reply:
So the check is only done partially here.. :)